receiver/redisreceiver/config.md

```diff
@@ -0,0 +1,267 @@
+# Redis Receiver Reference
+
+
+
+### redisreceiver-Config
+
+| Name | Field Info | Default | Docs |
+| ---- | --------- | ------- | ---- |
+| collection_interval |[time-Duration](#time-Duration)| 10s |  |
+| endpoint |string| <no value> |  |
```

Comment:
Should we create a follow up issue to improve docs coverage for configuration structs?

Reply:
Sure. And there are historical comments that will end up getting rendered that are no longer very useful, such as [this one](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/configtls.go#L60). These should be updated to be useful to the end-user, and developers should be made aware that there is a mechanism that publishes them.

Reply:
Agreed that cleaning up comments will be important

Reply:
Created this issue https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/13384